### DISCUSSION TITLE
クエスト「ワーグを殲滅せよ」を放棄できるようにしてほしい

### DISCUSSION THREAD

**Discussion by chikatei** [hengband/hengband#607] [Closed] [answered]
一部のキャラクター（種族アンドロイドなど）はワーグクエストを行うメリットが薄いので放棄したくなるときがありますが、現状は放棄が不可能になっています。その後に続く湖の洞窟と幽霊屋敷もそれなりに重要なクエストなので、ワーグクエストを無視することもなかなかできません。村長の家で受諾を断るなど何らかの手段でワーグクエストを放棄できるようになれば、上記のような場合に退屈な作業をスキップできてより快適になるのではないかと思います。

**Comment by deskull-m** [comment, marked as answer]
実装済ませましたので完了とさせていただきます。ありがとうございました。
